server/instance.go

```diff
@@ -235,6 +317,35 @@ func StopServer(id int) error {
 			return err
 		}
 	}
+	
+	logfile, _ := createLogFile(server)
+	
+	if runtime.GOOS == "windows" {		
+		logrus.Info("Remove Firewall Rules")
+		fwTcpArgs := []string{"advfirewall", "firewall", "del", "rule", "name=\"ACCSERVER_" + strconv.Itoa(server.Id) + "\""}
+		fwTcpCmd := exec.Command("netsh", fwTcpArgs...)
+		fwTcpCmd.Stdout = logfile
+		fwTcpCmd.Stderr = logfile
+		fwTcpErr := fwTcpCmd.Start()
```

Comment:
If a rule exists, it will still be added, resulting in two identical rules. 
Since rules are removed by name, all rules are removed, including duplicates. I've never had a case where I can't remove a rule, only a notification when no rule with that name exists.
If that happens, I wouldn't know what to do except display an error. Maybe an issue on the website, but I'm not really familiar with Vue.
Another option would be to prevent the Session from beeing stopped, but I'm not sure if this will be a great idea, since it may result in unstopable Sessions if the error could not be fixed by the user